app/rne-theme/theme.ts

```diff
@@ -38,7 +38,9 @@ const theme = createTheme({
     Text: (props, theme) => {
       const universalStyle = {
         color: props.color || theme.colors.grey5,
-        fontFamily: "SourceSansPro",
```

Comment:
Why was this removed?

Reply:
red screen of death complaining the font doesn't exist. tried adding `-Regular` but got same message.

Reply:
here is the issue, with the previous code: 

```
        fontWeight: props.bold ? "600" : "400",
        fontFamily: "SourceSansPro-Regular",
```

<img width="311" alt="image" src="https://user-images.githubusercontent.com/11156370/220771312-d2d806fa-3fef-4147-86af-9cd72ce8a03c.png">

Reply:
> here is the issue, with the previous code:
> 
> ```
>         fontWeight: props.bold ? "600" : "400",
>         fontFamily: "SourceSansPro-Regular",
> ```
> 
> <img alt="image" width="311" src="https://user-images.githubusercontent.com/11156370/220771312-d2d806fa-3fef-4147-86af-9cd72ce8a03c.png">

@UncleSamtoshi if it's not been used in the app but only storybook I propose you to approve so we can unlock the release and can dive into this whenever we use the component

Reply:
Works for me